bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
If we're going to get into this game...that's only going to work on `CONFIG_USER_NS=n` systems (Arch), not `CONFIG_USER_NS=y  # restrictions added` systems (Debian stable, RHEL7).

So a bubblewrap program (flatpak) that unconditionally adds `--unshare-user-try` will still fail on the latter systems, right?  In that case I'm not sure what we've gained with this patch.

A wrapper program can (without any privileges) just do this same stat.